### PR TITLE
[Maxwell]-Building-a-blog-subscription-dapp on-Celo-part-1

### PR DESCRIPTION
**Pull request by maxzysparks**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.
- [x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: src/data-tutorials/users.tsx [hunk 1]
```diff
@@ -2066,11 +2066,11 @@ const Users: User[] = [
...
-    title: 'Verifying Contracts with Hardhat',
-    description: 'Hardhat is one of the most popular developer tools for writing contracts for EVM compatible blockchains.',
-    preview: require('./showcase/intermediate/verifying-contracts-with-hardhat.png'),
-    website: '/blog/hardhat-deploy-verify',
-    tags: ['intermediate','foundation', 'hardhat', 'sourcify', 'verification'],
+    title: 'Building a blog subscription dapp on Celo part 1',
+    description: 'Learn how to build a Celo-based blog subscription dapp in this comprehensive guide.',
+    preview: require('./showcase/advanced/blog.png'),
+    website: '/blog/building-a-blog-subscription-dapp-on-celo-part-1',
+    tags: ['celo','advanced', 'solidity', 'celosage'],
```
Comment (4undRaiser) [inline src/data-tutorials/users.tsx:2073]: change the advanced tag to intermediate

File: blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-/Building-a-blog-subscription-dapp-on-Celo-part-1.md [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+  - name: Maxwell Onyeka
+    title: Technical Writer
+    url: https://github.com/maxzysparks
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celo, advanced, solidity, celosage]
+hide_table_of_contents: true
+slug: /tutorials/building-a-blog-subscription-dapp-on-celo-part-1
+---
+
+![header](../../src/data-tutorials/showcase/advanced/blog.png)
```
Comment (4undRaiser) [inline blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-/Building-a-blog-subscription-dapp-on-Celo-part-1.md:14]: change to intermediate

File: blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-/Building-a-blog-subscription-dapp-on-Celo-part-1.md [hunk 3]
```diff
@@ -0,0 +1,328 @@
+---
+title: Building a blog subscription dapp on Celo part 1
+description: Learn how to build a Celo-based blog subscription dapp in this comprehensive guide.
+authors:
+  - name: Maxwell Onyeka
+    title: Technical Writer
+    url: https://github.com/maxzysparks
+    image_url: https://avatars.githubusercontent.com/u/80678178?v=4
+tags: [celo, advanced, solidity, celosage]
```
Comment (4undRaiser) [inline blog/04-30-2023-Building-a-blog-subscription-dapp-on-Celo-part-/Building-a-blog-subscription-dapp-on-Celo-part-1.md:9]: change to intermediate

File: src/data-tutorials/users.tsx [hunk 4]
```diff
@@ -2066,11 +2066,11 @@ const Users: User[] = [
...
   },
   {
-    title: 'Verifying Contracts with Hardhat',
-    description: 'Hardhat is one of the most popular developer tools for writing contracts for EVM compatible blockchains.',
-    preview: require('./showcase/intermediate/verifying-contracts-with-hardhat.png'),
-    website: '/blog/hardhat-deploy-verify',
-    tags: ['intermediate','foundation', 'hardhat', 'sourcify', 'verification'],
+    title: 'Building a blog subscription dapp on Celo part 1',
+    description: 'Learn how to build a Celo-based blog subscription dapp in this comprehensive guide.',
+    preview: require('./showcase/advanced/blog.png'),
```
Comment (4undRaiser) [inline src/data-tutorials/users.tsx:2071]: after renaming the image to this building-a-blog-subscription-dapp-on-celo-part-1, move it into the intermediate folder in showcase and update it here by changing to intermediate

Comment (maxzysparks) [reply src/data-tutorials/users.tsx:2071]: the image name is fine as it is

**Pull request merged**